web/styles/subscriptions.css

```diff
@@ -497,7 +505,12 @@ h4.user_group_setting_subsection_title {
     width: 100%;
 }
 
-#clear_search_stream_name,
+#clear_search_stream_name {
+    position: absolute;
+    padding-top: 14px;
+    right: -20px;
+}
+
 #clear_search_group_name {
```

Comment:
Splitting this CSS is likely a mistake; group settings has the same component, so likely you should instead be fixing the same buig there.